wam2layers/preprocessing/checks.py

```diff
@@ -0,0 +1,146 @@
+import warnings
+from pathlib import Path
+
+import numpy as np
+import yaml
+
+
+def _warning_on_one_line(message, category, filename, lineno, file=None, line=None):
+    """A more compact warning format.
+
+    https://stackoverflow.com/a/26433913
+    """
+    short_path = Path(filename).relative_to(Path(filename).parents[2])
+    return f'{short_path}:{lineno}: {category.__name__}: {message}\n'
+
```

Comment:
I do not really get what this warning_on_one_line does? What is the use of this function

Reply:
This a bit technical indeed, I'd just skip over it if I were you. What it does: make the formatting of the warning messages a bit nicer. Now it prints something like: `wam2layers/preprocessing/checks.py:35: UserWarning: precip has no units`